src/web/topic/components/TopicWorkspace/MoreActionsDrawer.tsx

```diff
@@ -342,35 +343,37 @@ export const MoreActionsDrawer = ({
               </ToggleButton>
             </ListItem>
 
-            <ListItem disablePadding={false}>
+            <ListItem disablePadding>
               <Typography variant="body2">Layout Thoroughness</Typography>
```

Comment:
As mentioned here https://github.com/amelioro/ameliorate/pull/620/files#r1894997107, since we're using `Select` still, we should remove `Layout` to keep this text from having to wrap to two lines.

```suggestion
              <Typography variant="body2">Thoroughness</Typography>
```